test/driver_test.rb

```diff
@@ -129,8 +129,38 @@
       expect(@driver.average_rating).must_be_close_to (5.0 + 1.0) / 2.0, 0.01
     end
   end
-
+  #====================
   describe "total_revenue" do
```

Comment:
Include tests for the following edge cases: a driver with no trips and a driver with in progress trips.